PubSub/src/Topic.php

```diff
@@ -731,7 +786,6 @@ private function formatMessage($message)
     private function subscriptionFactory($name, array $info = [])
     {
         return new Subscription(
-            $this->connection,
```

Comment:
Because we are not passing in the `requestHandler`, the `Subscription` object would then create a _new_ `RequestHandler` class, which would then in turn create new `GapicClient` classes. This is inefficient and could result in unexpected behavior!

Reply:
Fixed that in the latest version. The GAPIC classes are now initialized in the resource classes itself(like Topic, Subscription etc) and the same instance is passed on to the different requests.

Thanks for pointing this out.